src/query/storages/common/cache/src/providers/table_data_cache.rs

```diff
@@ -0,0 +1,196 @@
+// Copyright 2023 Datafuse Labs.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::sync::Arc;
+use std::thread::JoinHandle;
+
+use common_cache::Count;
+use common_cache::DefaultHashBuilder;
+pub use common_cache::LruDiskCache as DiskCache;
```

Comment:
`use common_cache::LruDiskCache as DiskCache;`?

Reply:
DiskCache has been moved into `storage-common-cache` crate, and LruDisckCache is now a type alias of tDiskCache with some concrete type argument